Library/Formula/zero-install.rb

```diff
@@ -0,0 +1,152 @@
+class ZeroInstall < Formula
+  desc "Zero Install is a decentralised software installation system"
+  homepage "http://0install.net/"
+  url "https://downloads.sf.net/project/zero-install/0install/2.8/0install-2.8.tar.bz2"
+  sha256 "12de771be748bce9350c90bc4720029a566b078ceabd335af09386ac6a37df2b"
+
+  head do
+    url "https://github.com/0install/0install.git"
+    depends_on "gettext" => :build
+  end
+
+  option 'without-gui', "Skip the 0install gui (requires GTK+)"
+
+  depends_on :gpg
+  depends_on "curl"
+  depends_on "gtk+" if build.with? "gui"
```

Comment:
That is unfortunate (no gui bottle), since gtk+ is one of the more annoying ones.
But at least you can install *that* + deps (and ocaml/opam + boost) from bottles...